docs/getting-started.md

```diff
@@ -21,12 +21,24 @@ The [PayID Sandbox](https://payid.org/sandbox) lets you experiment with PayID in
 
 To use [PayID Sandbox](https://payid.org/sandbox), log in with your Github account and follow the prompts. You can quickly set up a virtual test server, and then create users with simple PayID addresses that map to addresses on various networks such as XRPL, Interledger, BTC, and ETH. Using the provided cURL or Javascript code, you can create, update, and delete users, and get payment information about a PayID address. While the session is running, you can also run operations from your command line.
 
-## Set up your own PayID server
+## Run a PayID Server
 
-You have several options to set up your own PayID server. See [Deployment Options](intro-deploy). For preliminary development and testing, try deploying locally, with or without Docker.
+To get started, the first thing to do is set up a local PayID server. Here is a simple method. For more options, see [Deployment Overview](intro-deploy).
+
+1.  If you haven't already, install [Docker](https://docks.docker.com/get-docker/) and [Node](https://nodejs.org/en/) on your machine.
+
+2.  The reference implementation server is available at [https://github.com/payid-org/payid](https://github.com/payid-org/payid). You can download the reference implementation server by cloning the repository at the command line:
+    `git clone https://github.com/payid-org/payid.git`
+
+3.  Go to the `/payid` directory that you just created:
+    `cd payid`
+
+4.  Boot up the PayID HTTP server and a Postgres database to develop against:
+    `npm run devEnvUp`
 
 ## Create a PayID
 
+Now that you have a PayID server running, you can create a PayID. If your server was publicly accessible, other people could query your account information from your PayID server.
```

Comment:
Does this duplicate the following paragraph? The tenses are different in both versions. I'd vote for present tense or whatever's most easily consistent.